src/cloudai/schema/core/test.py

```diff
@@ -13,9 +13,10 @@
 # limitations under the License.
 
 import sys
-from typing import Dict, List, Optional, Union
+from typing import TYPE_CHECKING, Dict, List, Optional, Union
 
-from .test_template import TestTemplate
+if TYPE_CHECKING:
+    from .test_template import TestTemplate
```

Comment:
Can you help me understand this? What is the main purpose of using TYPE_CHECKING when we can just import the module directly? I did not know there is such a constant.

Reply:
This is indeed not needed here. Left from my attempts to fix circular dependencies. Thanks for the catch!